src/RL/utils/geometricflux/heterogeneousgraphconvinit.jl

```diff
@@ -0,0 +1,43 @@
+struct HeterogeneousGraphConvInit{A <: AbstractMatrix, B}
+    weightsvar::A
+    weightscon::A
+    weightsval::A
+    biasvar::B
+    biascon::B
+    biasval::B
+    σ
+end
+
+function HeterogeneousGraphConvInit(in::Array{Int}, out::Int, σ=identity; init=Flux.glorot_uniform, bias::Bool=true, T::DataType=Float32)
```

Comment:
The base function is `σ=identity` following what has been done on [this paper.](https://www.sciencedirect.com/science/article/pii/S0952197621004255?via%3Dihub)